packages/inter-protocol/test/vaultFactory/vault-collateralization.test.js

```diff
@@ -15,6 +16,46 @@ test.before(async t => {
   trace(t, 'CONTEXT');
 });
 
+test('totalDebt calculation includes compoundedInterest', async t => {
+  const { aeth, run } = t.context;
+  const md = await makeManagerDriver(t);
+  const timer = md.timer();
+
+  t.log('charge 2% per day to simulate lower rates over longer times');
+  const aethKey = { collateralBrand: aeth.brand };
+  const twoPctPerDay = run.makeRatio(2n * 360n, 100n);
+  await md.setGovernedParam('InterestRate', twoPctPerDay, { key: aethKey });
+  const plausibleDebtLimit = run.units(500000);
+  await md.setGovernedParam('DebtLimit', plausibleDebtLimit, { key: aethKey });
+
+  t.log('mint 100 IST against 25 AETH');
+  const vd = await md.makeVaultDriver(aeth.units(25), run.units(100));
+  const v1debtAfterMint = await E(vd.vault()).getCurrentDebt();
+  t.deepEqual(v1debtAfterMint, run.units(100 + 5));
+  // totalDebt is debit of this 1 vault
+  await md.metricsNotified({ totalDebt: v1debtAfterMint }, AT_NEXT);
+
+  await E(timer).tickN(40, 'interest accumulates over 40hrs');
+  await eventLoopIteration(); // let all timer-induced promises settle
+  const v1debtAfterDay = await E(vd.vault()).getCurrentDebt();
+  t.deepEqual(v1debtAfterDay, run.make(106_008_000n));
+
+  // Checking that totalDebt here matches v1debtAfterDay would be handy,
+  // but totalDebt isn't updated when calculating interest.
+  const expectedCompoundedInterest = {
+    denominator: { value: 100000000000000000000n },
+    numerator: { value: 100960000000000000000n },
```

Comment:
all these zeros cancel out because it's a ratio
```suggestion
    denominator: { value: 10000n },
    numerator: { value: 10096n },
```

Reply:
the comparison is done with `t.like()`, which doesn't know that this is a ratio.

It might be nice if our ratio calculations always reduced common factors, but... they don't.